packages/babel-preset-env/data/built-ins.json

```diff
@@ -215,10 +215,10 @@
     "edge": "12",
     "firefox": "5",
     "safari": "10.1",
-    "node": "10.0",
+    "node": "10",
```

Comment:
I think that `10.0` was better? Other features (e.g. `proposal-async-generator-functions`) have `node: 10.13`, so the minor version could be meaningful.

Reply:
For me, it’s about being consistent, so we don’t have multiple ways Of representing the same thing (`4.0` for one, and `4` for another).

I don’t find `10` vs `10.13` hard to decipher, in fact, I think that it makes minor versions _more_ noticeable.

But subjective stuff aside, thoughts on at least being consistent? Ultimately I’m fine with reverting this... not a huge deal :)

Reply:
If we handle `10` exactly as `10.0` it's ok like you prefer, I don't really care about it